### PR TITLE
898 update readout item icon to accept a glyph

### LINKED ISSUE #898: Update ReadoutItemIcon to accept a Glyph as content
`ReadoutItemIcon` currently receives a string corresponding to an icon in the UIKit library. We need to be able to render an `Icon` containing a `Glyph` from our own library.

### PR DESCRIPTION
**Pull request by karinakeen**
This PR resolves #898 

@shanberg 

 I plugged in existing glyphs for viewing in these files: 

`/gm-fabric-dashboard/src/components/Main/scenes/Instance/scenes/go/scenes/Summary.js` and 
`gm-fabric-dashboard/src/components/Main/scenes/Instance/scenes/jvm/scenes/Summary.js`

To pass additional styling props to glyph, pass props to `ReadoutItemIcon`  and then to `Icon` on line 43 in `/gm-fabric-dashboard/src/components/Main/components/Readout/components/ReadoutItem/components/ReadoutItemIcon.js`:

Add new glyphs to this file `/gm-fabric-dashboard/src/components/Glyphs/index.js` then you can call individual glyph by calling `<Glyph name="glyphname"/>`

### CODE REVIEW

File: src/components/Glyphs/index.js [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
-// this dynamic  component is used for story testing only.
+import NoKey from "./NoKey";
+import NoMetrics from "./NoMetrics";
+import Pause from "./Pause";
+import Play from "./Play";
+import Poll from "./Poll";
+import RunningSmall from "./RunningSmall";
+import Summary from "./Summary";
+import Tape from "./Tape";
+import Threads from "./Threads";
```
Comment (tilleryd) [inline src/components/Glyphs/index.js]: There's some awesome alphabetization of imports here.

File: src/components/Main/scenes/Instance/scenes/jvm/scenes/Summary.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ function SummaryGrid({
           <Readout
             readoutItems={[
               {
-                icon: "server",
+                icon: "Tape",
```
Comment (tilleryd) [inline src/components/Main/scenes/Instance/scenes/jvm/scenes/Summary.js]: @karinakeen Are these glyphs just placeholders for @shanberg to replace?

Comment (karinakeen) [reply src/components/Main/scenes/Instance/scenes/jvm/scenes/Summary.js]: @tilleryd @shanberg yes, they are placeholders

**Comment by karinakeen** [comment]
@shanberg Stuart will adjust sizing and reopen PR

**Comment by karinakeen** [comment]
All changes implemented in PR 910.  Closing this PR.

**Pull request closed without merging**